lib/notifications/notification_server.ex

```diff
@@ -120,6 +139,37 @@ defmodule Notifications.NotificationServer do
     send(caller_id, {:new_notification, value})
   end
 
+  defp broadcast_notification_to_other_instances(
+         %Notifications.Notification{
+           id: notification_id,
+           content: %Notifications.Db.Detour{}
+         },
+         options \\ []
+       )
+       when not is_nil(notification_id) do
+    # Currently, we've implemented our own "PubSub" for notifications and we
+    # are not using the provided `Phoenix.PubSub` that comes with Phoenix
+    # channels. This means we don't benefit from Phoenix PubSub's ability to
+    # send messages using distributed Elixir, and that we need to implement
+    # this ourselves at this current time.
+    # Ideally, Notifications would be delivered using
+    # `Phoenix.Channel.broadcast` instead of our custom `broadcast` function
+    #  in `NotificationServer`. To do this, we'd need to implement the same
+    # filtering mechanism that this module has implemented. For now, we'll
+    # send messages to other Skate instances letting them know about new
+    # Notifications.
+
+    # Skate instances currently do not "specialize", and therefore we need to
+    # send the notification to all instances
+    for node <- Node.list() do
+      Node.spawn(node, fn ->
+        server = Keyword.get(options, :server, default_name())
```

Comment:
suggestion: you shouldn't need `Node.spawn` here. You can target the named server on another node with `{process_name, node}`: https://github.com/mbta/ride_along/blob/1d393329af28c00f080d37b730b1ad725e70a59d/lib/ride_along/rider_notifier.ex#L60
```elixir
server = Keyword.get(options, :server, default_name())
for node <- Node.list() do
  GenServer.cast({server, node}, {:detour_activated_notification, notification_id})
end
```

Reply:
Awesome, that works!
https://github.com/mbta/skate/compare/c9cf2debf1e309c4140e4b43753b2d8b26356a97..37026f15ff15bba582f0b85b6ad6e158bba2142d

If you have any thoughts on how to automatically test this, let me know? 
I was looking at https://elixirschool.com/en/lessons/advanced/otp_distribution#conditionally-excluding-tests-with-tags-11 but that didn't mention anything about automatically setting up a second test instance, and when I tried to do so, it seemed like I'd have to run a second real instance of Skate for the tests to run against.

Reply:
I don't have any good answers for this, unfortunately. For RideAlong, I tested the code paths manually and tried to keep them simple.

If you don't have something already, you could test calling the `cast` and making sure is has the right effect?

Reply:
>> If you have any thoughts on how to automatically test this, let me know?
>
> I don't have any good answers for this, unfortunately. For RideAlong, I tested the code paths manually and tried to keep them simple.

This is what I've been doing locally
```sh
PORT=4001 iex \
	--sname 1@localhost \
	--cookie test \
	-S mix phx.server
```
```sh
iex \
	--sname 2@localhost \
	--cookie test \
	-S mix phx.server
```
And then run this on `2@localhost` shell
```ex
Node.connect(:"1@localhost")
```
(which means I've not been able to test `DNSCluster`, because it gets mad when I don't run it in release mode, as well as I don't think I can put a socket address (IP + Port) into what would be returned from DNS for `localhost`)

<details>
<summary>extra</summary>

or of course the corresponding call for the other server
```ex
Node.connect(:"2@localhost")
```

</details>

And then I open a browser window to both servers and create and activate a detour and confirm that the other browser window on the other server.